src/appleseed/renderer/modeling/scene/scene.cpp

```diff
@@ -31,6 +31,7 @@
 #include "scene.h"
 
 // appleseed.renderer headers.
+#include "renderer/kernel/intersection/embreescene.h"
```

Comment:
Looks redundant. Remove?

Reply:
No, I need it here to return EmbreeDevice type

Reply:
I see.

Maybe it would make sense to split `embreescene.h` into `embreescene.h` and `embreedevice.h`.